engine/src/juliabox/plugins/auth_github/github_auth.py

```diff
@@ -58,8 +65,12 @@ def get(self):
         if code is not False:
             user = yield self.get_authenticated_user(redirect_uri=self_redirect_uri, code=code)
             user_info = yield self.get_user_info(user)
-            self.update_user_profile(user_info)
             user_id = user_info['email']
+            if not user_id:
+                self.rendertpl("index.tpl", cfg=JBoxCfg.nv, state=self.state(
+                    error="Please make your email public in your GitHub profile if you wish to use GitHub Authentication", success="")
```

Comment:
"to sign in with GitHub" seems better to me instead of "to use GitHub Authentication"